ENIGMAsystem/SHELL/Platforms/General/PFjoystick.h

```diff
@@ -21,6 +21,11 @@
 
 #include <string>
 
+namespace enigma {
+  void init_joysticks();
+  void handle_joysticks();
```

Comment:
I don't know if I agree with these being declared here. If you are going to do it anyway at least delete the old declarations of them wherever they were.

Reply:
If I were to go the extra mile and implement BSD joysticks, there is a good chance it will need initialization and handling code, and because OS X is based on BSD and both being Unix like as Linux is, we don't have any real way of knowing outside of an educated guess that Windows is *probably* the only platform which won't utilize these two functions, and that's completely ignoring the fact we are planning to do mobile as well which could potentially need these.